autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -100,15 +93,6 @@ class TabularFeatureValidator(BaseFeatureValidator):
         categorical_columns (List[int]):
             List of indices of categorical columns
     """
-    def __init__(
-        self,
-        logger: Optional[Union[PicklableClientLogger, Logger]] = None,
-        dataset_compression: Optional[Mapping[str, Any]] = None,
-    ) -> None:
-        self._dataset_compression = dataset_compression
-        self._reduced_dtype: Optional[DatasetDTypeContainerType] = None
-        super().__init__(logger)
```

Comment:
Can we keep `__init__` as we need it for dtypes provided by users.?

Reply:
okay I'll add it back